webdriver-spec.html

```diff
@@ -3237,183 +3237,428 @@ <h2 id=element-displayedness>Element Displayedness</h2>
       What happens to unhandled JS errors? Caused by a user's JS? Caused by JS
       on a page? How does a user of the API obtain the list of errors? Is that
       list cleared upon read?
-    </div>
-    <p class="note">Conformance tests for this section can be found in the <a href="https://github.com/w3c/web-platform-tests/tree/master/webdriver/">webdriver module</a> under the "ecmascript" folder.</p>
+  </div>
 
-    <section>
-      <h2>Javascript Command Parameters</h2>
+  <p>When required to <dfn>JSON deserialize</dfn> with
+  argument <var>value</var>, and optional argument <var>seen</var>, a
+  remote end must run the following steps:</p>
+  <ol>
+    <li><p>If <var>seen<var> was not provided, let <var>seen</var> be
+    an empty set.</p></li>
+    <li><p>Jump to the first appropriate step below:
+      <dl>
+        <dt>If <var>value</var> is <code>null</code>, or has
+        type <code>Boolean</code>, <code>Number</code>
+        or <code>String</code>:</dt>
+        <dd><p>Return <a>success</a> with data <var>value</var></p></dd>
+
+        <dt>If <var>value</var> is an <code>Object</code>
+        that <a>represents a web element</a>:</dt>
+        <dd><p>Return the result of running the <a>deserialize a web
```

Comment:
s/a/the/ to match the **deserialise the web element** algorithm, or use the `title` attribute